src/amp/components/Ad.tsx

```diff
@@ -107,22 +110,26 @@ interface CommercialConfig {
 }
 
 export const Ad = ({
+	isSticky,
 	adRegion,
 	edition,
 	section,
 	contentType,
 	config,
 	commercialProperties,
 }: {
+	isSticky?: boolean;
 	adRegion: AdRegion;
 	edition: Edition;
 	section: string;
 	contentType: string;
 	config: CommercialConfig;
 	commercialProperties: CommercialProperties;
 }) => {
-	const { width, height } = sizes[0]; // Set initial size to MPU
-	const multiSizes = sizes.map((e) => `${e.width}x${e.height}`).join(',');
+	const adSizes = isSticky ? stickySizes : inlineSizes;
+	const [{ width, height }] = adSizes; // Set initial size as first element (should be the largest)
```

Comment:
```suggestion
	const [{ width, height }] = adSizes[0]; // Set initial size as first element (should be the largest)
```

Reply:
We're already accessing the first element of the array via destructuring.

`const [{ width, height }] = adSizes[0]` is the same as `const { width, height } = adSizes[0]`

Reply:
Sorry, I didn't spot those [square brackets]!